mlflow/server/handlers.py

```diff
@@ -2437,6 +2439,40 @@ def _delete_trace_tag(request_id):
     return _wrap_response(DeleteTraceTag.Response())
 
 
+@catch_mlflow_exception
+@_disable_if_artifacts_only
+def get_trace_artifact_handler():
+    from querystring_parser import parser
+
+    query_string = request.query_string.decode("utf-8")
+    request_dict = parser.parse(query_string, normalized=True)
+    request_id = request_dict.get("request_id")
+
+    if not request_id:
+        raise MlflowException('Request must include the "request_id" query parameter.')
```

Comment:
```suggestion
        raise MlflowException('Request must include the "request_id" query parameter.', error_code=...)
```

can we set the appropriate error code?